addons/website_sale/views/res_config_settings_views.xml

```diff
@@ -185,10 +185,10 @@
                         id="promotion_coupon_programs"
                         title="Boost your sales with multiple kinds of programs: Coupons, Promotions, Gift Card, Loyalty. Specific conditions can be set (products, customers, minimum purchase amount, period). Rewards can be discounts (% or amount) or free products.">
                         <div class="o_setting_left_pane">
-                            <field name="module_sale_loyalty" />
```

Comment:
I don't think you need to change that either, just putting sale_loyalty and pos_loyalty in auto_install = True should do the job.

Because:
if you have sale and a point of sale installed, but not yet loyalty. When you'll install pos_loyalty (for example) it will install 'Loyalty' as it is a dependency.
An then, as loyalty is installed and sales too, sale_loyalty will be installed as it is auto_install.

Reply:
I am responding also here for the sake of completeness: 
The reason we changed this field is in order to have all the "Loyalty Settings" from all 3 apps (PoS, eCommerce & Sales) in one common setting, so they change synchronously. Video demo: https://watch.screencastify.com/v/VoWzp3H1hEB9K84dNO3n